k8s.gcr.io/images/k8s-staging-apisnoop/images.yaml

```diff
@@ -0,0 +1,13 @@
+# These images do not exist yet
+- name: auditlogger
+  dmap:
+    "sha256:46b237764fee52288a7ab075353c77d4e0de55e374c87474409091bd94e442af": ["v1.0.0"]
```

Comment:
Let's drop the SHAs until images actually exist.

Reply:
:heavy_check_mark: 
Removed, left a placeholder comment.